src/chem/airnow2ioda-nc.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
```

Comment:
I think this is supposed to stay at python3 for compatibility issues on certain platforms.